src/temaWeek7Comparable/Persoane.java

```diff
@@ -1,5 +1,6 @@
 package temaWeek7Comparable;
 
+
```

Comment:
It would have been better to have the main class in a different class and keep the Persoane class clean; to serve just the purpose of describing a person.